polkadot-parachains/parachains-common/Cargo.toml

```diff
@@ -14,18 +14,18 @@ codec = { package = 'parity-scale-codec', version = '2.3.0', features = ['derive
 scale-info = { version = "1.0.0", default-features = false, features = ["derive"] }
 
 # Substrate dependencies
-sp-consensus-aura = { git = 'https://github.com/paritytech/substrate', branch = "master", default-features = false }
-sp-std = { git = 'https://github.com/paritytech/substrate', branch = "master", default-features = false }
-sp-io = { git = 'https://github.com/paritytech/substrate', branch = "master", default-features = false }
-frame-executive = { git = 'https://github.com/paritytech/substrate', branch = "master", default-features = false }
-frame-support = { git = 'https://github.com/paritytech/substrate', branch = "master", default-features = false }
-frame-system = { git = 'https://github.com/paritytech/substrate', branch = "master", default-features = false }
-pallet-asset-tx-payment = { git = 'https://github.com/paritytech/substrate', branch = "master", default-features = false }
-pallet-assets = { git = 'https://github.com/paritytech/substrate', branch = "master", default-features = false }
-pallet-authorship = { git = 'https://github.com/paritytech/substrate', branch = "master", default-features = false }
-pallet-balances = { git = 'https://github.com/paritytech/substrate', branch = "master", default-features = false }
-sp-runtime = { git = 'https://github.com/paritytech/substrate', branch = "master", default-features = false }
-sp-core = { git = 'https://github.com/paritytech/substrate', branch = "master", default-features = false }
+sp-consensus-aura = { git = 'https://github.com/paritytech/substrate', default-features = false , branch = "master" }
```

Comment:
Hmm? :P

Reply:
This is actually because of diener. If you change a branch to Foo and revert to master, it reverts back with this format. Maybe you should patch it not to :D